code/game/objects/items/weapons/storage/mre.dm

```diff
@@ -171,6 +171,20 @@ MRE Stuff
 /obj/item/storage/box/fancy/mre/menu11/special // antag item
 	meal_desc = "This one doesn't have a menu listing. How odd. It has the initials \"A.B.\" written on the back."
 
+/obj/item/storage/box/fancy/mre/menu12_kois
```

Comment:
```suggestion
/obj/item/storage/box/fancy/mre/menu12
```

just call it menu 12, like all of the other MREs

Reply:
I'd agree with you on giving them proper names, but not here where it's just the one.

If you were to add proper names to all the paths, then I'd accept. Otherwise, none at all.

Reply:
keeping it as just menu12 for now